gensim/models/sldamodel.py

```diff
@@ -0,0 +1,223 @@
+import logging
+import numbers
+import os
+
+import numpy as np
+import six
+from scipy.special import gammaln, psi  
+from scipy.special import polygamma
+from six.moves import xrange
+from collections import defaultdict
+
+from gensim import interfaces, utils, matutils
+from gensim.matutils import dirichlet_expectation
+from gensim.models import basemodel, CoherenceModel
+
+try:
+    from scipy.special import logsumexp
+except ImportError:
+    from scipy.misc import logsumexp
+    
+logger = logging.getLogger('gensim.models.sldamodel')
+
+def update_dir_prior(prior, N, logphat, rho):
+    """
```

Comment:
please use [numpy-style](https://github.com/RaRe-Technologies/gensim/wiki/Developer-page#docstrings) docstrings (here and everywhere).

Reply:
Roger